### ISSUE TITLE
AvP AI index different from Alienness

### ISSUE THREAD

**Issue by lyy005** [GDKO/AvP#8]
Hi there,
Thank you for making this amazing tool to detect HGTs. I have a quick question about the AI index calculated from AvP. I'm looking for horizontally transferred genes in aphids. Potential donors of interest are bacteria, fungi, and viruses. So I set the groups.yaml as follows. Ingroups are metazoa. And I wanted to exclude all the hits to arthropods. 

`---`
`Ingroup:`
` 33208: Metazoa`

`EGP:`
` 6656: Arthropoda`

The resulting AI from AvP was 460.51701859880916, indicating it's likely a horizontally transferred gene. 

Then I used the same input BLAST result and same setting for Alienness as in AvP, i.e. ingroups are metazoa. And I wanted to exclude all the hits to arthropods. However, based on Alienness results, the same gene is not an HGT. 

Here're the parameters I used for Alienness. 

Parameters
-Taxon group of interest: 33208-Metazoa;
-Taxon group(s) to exclude: 6656-Arthropoda; 
-Taxon group(s) to classify: no taxa

I was wondering why the AI from Alienness and AvP are different? Did I set the AvP groups.yaml correctly? 

Thank you for any suggestions! 

YY

**Comment by GDKO** [comment]
Hi YY,

I am glad the tool is useful for your research. An AI that high indicates there are no hits from ingroup sequences. Both tools use the same AI calculation defined in [Gladyshev et al.](https://www.science.org/doi/10.1126/science.1156407) and you have specified the parameters correctly in both programs so the AI calculations should be identical. However, Alienness defines queries with ```AI > 15``` and ```percentage of identity > 70``` as likely contamination, so your query may be present in that category and not in the likely_hgt. 

Can you check the output files of Alienness whether that is the case?

Cheers,
Georgios

**Comment by lyy005** [comment]
Dear Georgios,
Thank you for the quick response. I checked the Alienness output. However, I didn't see the input sequence. I think Alienness does not include the sequence if the sequence is not classified as one of the three categories ("likely hgt", "possible hgt", or "likely contamination"). Attached are the diamond outputs I used for AvP and Alienness just in case it would be helpful. 

The protein "XP_016660728.1" was an aphid protein that is not supposed to be a horizontally transferred gene. And I mapped the protein to NR database using Diamond with the following parameter:  

For AvP: 
`diamond blastp -d nr_v20220917.dmnd -q XP_016660728.1.fasta -o XP_016660728.1.out --evalue 1e-3 --threads 40 --mid-sensitive -k 500 --outfmt 6 qseqid sseqid pident length mismatch gapopen qstart qend sstart send evalue bitscore staxids`

For Alienness: 
`diamond blastp -d nr_v20220917.dmnd -q XP_016660728.1.fasta -o XP_016660728.1.out --evalue 1e-3 --threads 40 --mid-sensitive -k 500 --outfmt 6`

[AvP.XP_016660728.1.diamond.out.gz](https://github.com/GDKO/AvP/files/10664659/AvP.XP_016660728.1.diamond.out.gz)  

[Alienness.XP_016660728.1.diamond.out.gz](https://github.com/GDKO/AvP/files/10664655/Alienness.XP_016660728.1.diamond.out.gz)

Thank you! 

YY
